en/sql/schema/index_stmt.rst

```diff
@@ -15,21 +15,33 @@ For how to use indexes on the **SELECT** statement like Using SQL Hint, Descendi
 
 ::
 
-    CREATE [UNIQUE] INDEX index_name ON [schema_name.]table_name <index_col_desc> ;
+    CREATE [UNIQUE] INDEX index_name ON [schema_name.]table_name <index_col_desc2> ;
      
-        <index_col_desc> ::=
-            { ( column_name [ASC | DESC] [ {, column_name [ASC | DESC]} ...] ) [ WHERE <filter_predicate> ] | 
-            (function_name (argument_list) ) }
-                { [[WITH ONLINE [PARALLEL parallel_count]] | [INVISIBLE] | [VISIBLE]] }
-                [COMMENT 'index_comment_string']
+        <index_col_desc2> ::=
+                  {
+                     (  {column_name | function_name (argument_list)} [ASC | DESC]
+                       [{, {column_name | function_name (argument_list)} [ASC | DESC]} ...] )
+                  }
+                  [WHERE <filter_predicate> ]
+                  [WITH <index_with_clause> [{, <index_with_clause>]} ...]
+                  [INVISIBLE]
+                  [COMMENT 'index_comment_string’]
+
+             <index_with_clause> ::= {ONLINE [PARALLEL parallel_count]} | <index_with_option>
+             <index_with_option> ::= {DEDUPLICATE ‘=‘ deduplicate_level}
 
 *   **UNIQUE**: creates an index with unique values.
 *   *index_name*: specifies the name of the index to be created. The index name must be unique in the table.
 
 *   *schema_name*: Specifies the schema name. If omitted, the schema name of the current session is used.
 *   *table_name*: specifies the name of the table where the index is to be created.
 *   *column_name*: specifies the name of the column where the index is to be applied. To create a composite index, specify two or more column names.
-*   **ASC** | **DESC**: specifies the sorting order of columns. 
+*   **ASC** | **DESC**: specifies the sorting order of columns.
+*   *deduplicate_level*: Specifies the deduplicate level (0 to 14). For details, see `DEDUPLICATE`_.
+
+.. note::
+
+    *deduplicate_level* is an integer from 0 to 14. 0 means an index with the same configuration as CUBRID 11.2 or earlier without the **DEDUPLICATE** option.
```

Comment:
```suggestion
    *deduplicate_level* is an integer from 0 to 14. 0 means an index with the same configuration as CUBRID 11.2 or earlier which doesn't support the **DEDUPLICATE** option.
```

Reply:
@mhoh3963 
This sentence will be corrected to accommodate hgryoo's comment above.